test/conformance/testdata/conformance.yaml

```diff
@@ -704,6 +704,13 @@
     the Job MUST execute to completion.
   release: v1.16
   file: test/e2e/apps/job.go
+- testname: ReplicaSet, is created, Replaced and Patched
+  codename: '[sig-apps] ReplicaSet Replace and Patch tests [Conformance]'
+  description: Create a ReplicaSet (RS) with a single Pod. The Pod MUST be verifiy
+    that it is running. The RS MUST scale to two replican and verify the scale count
```

Comment:
ugh, we missed a typo ... `replican` -> `replicas`

Reply:
Very sharp eye Dims!
I'll fix it quickly.

Reply:
Updated, Thank Dims